src/components/MapDetail/MapDetail.js

```diff
@@ -77,17 +77,38 @@ export default function MapDetail() {
             Falkland Islands
           </HeadingTwo>
           <Box align='center' direction='row' justify='between'>
-            <Box direction='row' gap='small'>
-              <Uppercase color='kelp'>51&#176;42'S 57&#176;51'W</Uppercase>
-              <Uppercase color='kelp'>3492 SQ MI / 9044 SQ KM</Uppercase>
+            <Box direction='row' gap='xsmall'>
+              <Uppercase color='kelp' size='0.75rem'>51&#176;42'S 57&#176;51'W</Uppercase>
+              <Uppercase color='kelp' size='0.75rem'>3492 SQ MI / 9044 SQ KM</Uppercase>
             </Box>
-            <CheckBox label={<Uppercase color='kelp'>Subject Grid</Uppercase>} />
+            <CheckBox label={<Uppercase color='kelp' size='0.75rem'>Subject Grid</Uppercase>} />
+          </Box>
+          <Box
+            align='center'
+            background='gray'
+            border={{ color: 'kelp' }}
+            flex
+            justify='center'
+            style={{ position: 'relative' }}
+          >
+            <Map 
+              bounds={[coordinates.southWest, coordinates.northEast]}
```

Comment:
I'm getting an `Uncaught Error: Bounds are not valid.` error preventing the MapDetail/Default storybook from rendering? I think from this line, but not entirely sure.

Reply:
Good catch. I didn't have the appropriate default props for the component.